src/components/App.js

```diff
@@ -1,23 +1,20 @@
 import React, { useEffect } from 'react';
-import 'src/components/app.css';
-import AddTaskInput from "src/components/AddTask/AddTask.js";
-import TaskList from 'src/components/TaskList/TasksList.js';
-import { useDispatch } from 'react-redux';
-import { fetchTasks } from 'src/components/TodoListSlice';
 
-function App() {
+import 'src/components/app.css';
+import ConnectedAddTaskInput from 'src/container/AddTask'
+import ConnectedTaskList from 'src/container/TaskList';
 
-  const dispatch = useDispatch();
+function App( {fetchTasks} ) {
```

Comment:
we dont need to fetchTasks() in whole app as it will render the whole app again and again. fetch todos inside tasksList component as it is resposible to display the todos